Automattic/wp-calypso
# Update plugin action modals

Fixes https://github.com/Automattic/dotcom-forge/issues/9728

## Proposed Changes

| Before  | After |
| :-------------: | :-------------: |
| <img src="https://github.com/user-attachments/assets/a025ada4-49b5-4f69-be78-99e3d0f6f2a2">  | <img src="https://github.com/user-attachments/assets/8640449f-b094-47af-befe-02303ebad0c2">|

1. Changed the title as well because there are lots of plugins with long names, for example:
![Header flooded@2x](https://github.com/user-attachments/assets/e9fb97cd-bdb4-4c1b-ba7c-691d7329600b)
2. Updated the action buttons

## Why are these changes being made?


* Align design more with Core Modal

## Testing Instructions



* Go to /plugins/manage
* Try all actions:
  * Update
  * Activate/Deactivate
  * Enable auto-update/Disable auto-update
  * Remove
* Test with the sites-per-plugin Modal as well
* Test in calypso and jetpack cloud

> [!NOTE]  
> If you clicked around and see that the action menu is messed up, that's a bug to be addressed in a separate PR as production currently has the same behavior. Opened issue https://github.com/Automattic/dotcom-forge/issues/9995
 to track.


https://github.com/user-attachments/assets/4b8869cb-80d0-4c2d-a097-7baae7a6f9f1



## Pre-merge Checklist
